docker/addons/cassandra-writer/channels.toml

```diff
@@ -0,0 +1,4 @@
+# If you want to listen on all channels, just pass one element ["*"], otherwise
+# pass the list of channels.
+[channels]
```

Comment:
Why do we need a special table for this config? Would there be any other setting for channels except `filter`? I.e. would just
```
channels = [...]
```
do the job?

Moreover, if all that we need is alist of channels, maybe we do not need even TOML? Maybe a simplle file with one channel ID per line is enough?

Reply:
We talked about adding support for configuration for every service (instead of using current env vars). That's why I added it as a separate table (this config will grow, and in the end, this config will create a lot of other fields). We can use simple file, but I would keep standard config format (especially because this config will grow).

Reply:
Agreed.